File: TalentAdmin/App_Code/ErrorObj.vb
Public Class ErrorObj

    Private _hasError As Boolean = False            ' true false flag

    Private _errorMessage As String = String.Empty  ' System generated error message
    Private _errorNumber As String = String.Empty   ' program generated error number
    Private _errorInfo As String = String.Empty
    Private _errorLogFile As String = "C:/DBAdminError.log"

    Public Property HasError() As Boolean
        Get
            Return _hasError
        End Get
        Set(ByVal value As Boolean)
            _hasError = value
        End Set
    End Property

    Public Property ErrorMessage() As String
        Get
            Return _errorMessage
        End Get
        Set(ByVal value As String)
            _errorMessage = value
        End Set
    End Property
    Public Property ErrorNumber() As String
        Get
            Return _errorNumber
        End Get
        Set(ByVal value As String)
            _errorNumber = value
        End Set
    End Property
    Public Property ErrorInfo() As String
        Get
            Return _errorInfo
        End Get
        Set(ByVal value As String)
            _errorInfo = value
        End Set
    End Property
    Public Property ErrorLogFile() As String
        Get
            Return _errorLogFile
        End Get
        Set(ByVal value As String)
            _errorLogFile = value
        End Set
    End Property

End Class

File: TalentSystemDefaults/TalentSystemDefaults/App_Code/ErrorObj.cs
using System;
namespace TalentSystemDefaults
{
	[Serializable()]
	public class ErrorObj
	{
		private bool _hasError = false; // true false flag
		private string _errorMessage = string.Empty; // System generated error message
		private string _errorNumber = string.Empty; // program generated error number
		private string _errorStatus = string.Empty; // program generated error code
		private string[] _itemErrorCode = new string[501]; // Error code for multiple items (e.g orders) per transaction
		private string[] _itemErrorMessage = new string[501]; // Error message for multiple items (e.g orders) per transaction
		private string[] _itemErrorStatus = new string[501]; // Error status for multiple items (e.g orders) per transaction
		public bool HasError
		{
			get
			{
				return _hasError;
			}
			set
			{
				_hasError = value;
			}
		}
		public string ErrorMessage
		{
			get
			{
				return _errorMessage;
			}
			set
			{
				_errorMessage = value;
			}
		}
		public string ErrorNumber
		{
			get
			{
				return _errorNumber;
			}
			set
			{
				_errorNumber = value;
			}
		}
		public string ErrorStatus
		{
			get
			{
				return _errorStatus;
			}
			set
			{
				_errorStatus = value;
			}
		}
		public string get_ItemErrorCode(int item)
		{
			return _itemErrorCode[item];
		}
		public void set_ItemErrorCode(int item, string value)
		{
			_itemErrorCode[item] = value;
		}
		public string get_ItemErrorMessage(int item)
		{
			return _itemErrorMessage[item];
		}
		public void set_ItemErrorMessage(int item, string value)
		{
			_itemErrorMessage[item] = value;
		}
		public string get_ItemErrorStatus(int item)
		{
			return _itemErrorStatus[item];
		}
		public void set_ItemErrorStatus(int item, string value)
		{
			_itemErrorStatus[item] = value;
		}
	}
}
